### DISCUSSION TITLE
Upgrade vals version to v0.18.0 / support tfstateazurerm subdir lookup

### DISCUSSION THREAD

**Discussion by Amos-85** [helmfile/helmfile#157] [Questions] [answered]
Since `vals` depends on `tfstate-lookup`, need to upgrade `vals` version [v0.18.0](https://github.com/variantdev/vals/issues/79) in order to include `tfstateazurerm subdir lookup` for the next helmfile release

References:
[vals bug fix](https://github.com/variantdev/vals/pull/88)
[tfstate-lookup bug fix](https://github.com/fujiwara/tfstate-lookup/pull/43)

**Comment by yxxhero** [comment, marked as answer]
see: https://github.com/helmfile/helmfile/pull/158
done

**Reply by Amos-85** [reply]
Thanks @yxxhero
